stdlib/public/core/Character.swift

```diff
@@ -353,6 +353,14 @@ public struct Character :
   internal var _representation: Representation
 }
 
+extension Character : CustomStringConvertible {
```

Comment:
The seemingly vacuous conformances of `Character` and `String` are meant to support uses like:

```swift
struct Passport {
    var number: Int64
    var genderMarker: Character
    var surname: String
    …
}

passport.number = try prompt("Passport number?")
passport.genderMarker = try prompt("Gender marker?")
passport.surname = try prompt("Last name?")

func prompt<T: LosslessStringConvertible>(_ message: String) throws -> T {
    while true {
        print(message, terminator: "")
        guard let line = readLine() else { throw CustomsCheckpointError.noInput }
        if let value = T(line) { return value }
        print("I didn't understand; please try again.")
    }
}
```

Or (if it wants to go to the trouble of escaping and unescaping strings):

```swift
extension Range: LosslessStringConvertible where Bound: LosslessStringConvertible { … }
```

The conformances are trivial, but they help make code which is generic on LosslessStringConvertible more useful.